packages/fast-application-utilities/README.md

```diff
@@ -1,4 +1,5 @@
 # FAST application utilities
+**FAST application utilities has been deprecated. Use `@microsoft/fast-web-utilities` instead.**
```

Comment:
Could we also include a link to the recommended package readme.md?

Reply:
@awentzel how about the npm package? That makes more sense to me.